scripts/pin_the_slow_way.py

```diff
@@ -37,14 +37,16 @@
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
           'libnetcdf': 'libnetcdf 4.4.*',
-          'libpng': 'libpng >=1.6.21,<1.7',
+          'libpng': 'libpng 1.6.23',
```

Comment:
Not sure to be honest. But I got some reports of breaks in libpng `1.6.21` (conda-forge) and `1.6.22` (`defaults`)